tests/element_test.py

```diff
@@ -1,12 +1,13 @@
 u"""Tests for Crystal and CrystalSlice
 """
 from ast import Assert
+from re import L
```

Comment:
These two imports should be qualified:

import ast
import re

and then refer to re.L and ast.Assert. It's very hard to know when you see a single letter "L" or Assert where it is coming from without that context.